clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -4240,15 +4242,34 @@ scheduler_tmpl_t<RecordType, ReaderType>::print_queue_stats()
         unsched_size = unscheduled_priority_.queue.size();
     }
     int live = live_input_count_.load(std::memory_order_acquire);
-    VPRINT(this, 1, "inputs: %zd scheduleable, %zd unscheduled, %zd eof\n",
-           live - unsched_size, unsched_size, inputs_.size() - live);
+    // Make our multi-line output more atomic.
+    std::ostringstream ostr;
+    ostr << "Queue snapshot: inputs: " << live - unsched_size << " scheduleable, "
```

Comment:
s/scheduleable/schedulable/?

Reply:
Hmm, seeing some hits of both, but there are more in this repo w/o the e.  OK, changed this and another one.